packages/gatsby-theme-carbon/src/components/Switcher/switcher.module.scss

```
@@ -0,0 +1,63 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~@carbon/elements/scss/type/type';
+
+.nav {
+  color: white;
```

```suggestion
  color: $carbon--white-0;
```

just in case... we ever change the color of white? 🤣